### ISSUE TITLE
Buttons vanish at larger zoom levels and on mobile

### ISSUE THREAD

**Issue by karlgroves** [WordPress/gutenberg#15347]
Buttons vanish at larger zoom levels and on mobile

-   **Severity:** High
-   **Affected Populations:**
    -   Low-Vision
    -   Cognitively Impaired
-   **Platform(s):**
    -   All / Universal
-   **Components affected:**
    -   Editor Top Bar
    -  Structure and Navigation

#### Issue description

Users relying on browser zoom to view the page content will eventually
trigger a mobile style at higher zoom levels. This mobile style hides
many of the buttons in the Editor Top Bar. There doesn't appear to be
another method of performing the Undo, Redo, Content Structure or Block
Navigation functions when the mobile view is triggered. Users with
keyboards may never see the shortcuts for using these controls, while
touchscreen users cannot use them at all.

The ability to zoom content may be essential for users with low-vision,
and may be helpful for users who have a cognitive disability. Avoiding
the need to scroll in the reading direction is also important, since
this significantly increases the effort required to read the content.

##### Issue Code

```
    .edit-post-header-toolbar>.components-button {


        display: none;


    }

```

#### Remediation Guidance

Find an equivalent way for low-vision desktop users (and mobile users)
to be aware of and use these controls, such as adding them under another
menu.

#### Relevant standards

-   [1.4.10 Reflow (Level
    AA)](https://www.w3.org/TR/WCAG20/#visual-audio-contrast-without-color)

**Note:** This issue may be a duplicate with other existing accessibility-related bugs in this project. This issue comes from the Gutenberg accessibility audit, performed by [Tenon](https://www.tenon.io) and funded by [WP Campus](https://wpcampus.org/). This issue is GUT-3 in Tenon's report

**Comment by mapk** [comment]
In [today's triage on slack](https://wordpress.slack.com/archives/C02S78ZAL/p1557498443240100), there's agreement that some design work should be done to retain these buttons on mobile. I'm swapping out labels to get some design work here.

**Comment by afercia** [comment]
Discussed during today's accessibility bug scrub, fully agree a more modern approach to responsive design for small screens would be not hiding important features 🙂

**Comment by melchoyce** [comment]
We should collapse them into a menu.

Two ideas...

**Select Menu**

![image](https://user-images.githubusercontent.com/2846578/69998675-d5e85300-1524-11ea-832e-87e4c4068b8c.png)

**DropdownMenu with an icon**

![image](https://user-images.githubusercontent.com/2846578/69998825-3aa3ad80-1525-11ea-987c-577c4b98ad3b.png)

**Comment by karmatosed** [comment]
Can I ask what is under tools @melchoyce? Right now with toolbar iterations, I see the following:

![image](https://user-images.githubusercontent.com/253067/70471252-b02ef100-1ac4-11ea-9bac-fb336c7034a2.png)

**Comment by melchoyce** [comment]
Is that in master? If so, this issue is probably fixed.

**Comment by karmatosed** [comment]
It is master. Considering that, let's close this as we can always reopen if it's still an issue.
